2022_01_03.md

```diff
@@ -110,6 +110,32 @@ cost and availability.
 AWS offers RISC-V instance types.
 </td>
 </tr>
+  
+  
+<tr>
+<td>
+  <a href="https://twitter.com/AaronDGoldman">@AaronDGoldman</a><br>
```

Comment:
I would also link to my github name to match the location of the written prediction in addition to twitter.
https://github.com/aarongoldman

This captures the prediction and is a cleaner wording of it.
Thanks for making me sound good.